src/misosoupy/psychopy_refine_item_list.py

```diff
@@ -45,22 +47,14 @@ def function_present_refined_item_list(
 
     Parameters
     ----------
+    num_items_to_select: int
+        Number of sounds necessary for experiment, i.e., up to how many sounds participants rank.
+        Defined in config.ini, updated in misosoupy.py if multiple categories are selected for 
+        (e.g., trigger and neutral) and there are fewer than num_items_to_select options available.
     mean_length : int
         Average number of characters comprising the sound labels. Used to determine font size.
     setup_item_height : int
         Font height. Default is 0.085.
```

Comment:
I think I would either specify that this default is defined in the config.ini file or put a default value here in the function too